README.md

```diff
@@ -88,7 +88,7 @@ make helm-chart-push
 ```bash
 kubectl create -f hello-world-read-module.yaml -n m4d-system
 ```
-3. Check if `M4DApplication` successfully deployed:
+3. Check if `FybrikApplication` successfully deployed:
 ```bash
 kubectl get m4dmodule hello-world-read-module -n m4d-system
 kubectl describe m4dmodule hello-world-read-module -n m4d-system
```

Comment:
Should the `m4dmodule` be changed to `fybrikmodule`? Also for `m4d-system` to `fybrik-system`?

Reply:
Thanks Fixed